packages/reporting-e2e/docker/reporting-server/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.8.11-buster
```

Comment:
This appears to be duplicated in `example-deployment`. Maybe have the example deployment base off this image?

Reply:
I'll delete the `dockerfile` from the `example-deployment` and I'll point directly to the dockerfile from the e2e-dashboard. So we can have just one. I'll update the documentation and the deploy script as well.